spec/index.bs

```diff
@@ -2653,6 +2676,41 @@ dictionary GPUFenceDescriptor : GPUObjectDescriptorBase {
 </script>
 
 
+Queries {#queries}
+================
+
+## QuerySet ## {#queryset}
+
+<script type=idl>
+interface GPUQuerySet {
+    GPUQueryType getQueryType();
+    GPUSize32 getQueryCount();
```

Comment:
I don't think these are needed; they're just returning user provided values.
If we want them, they should be `readonly attribute`s instead of methods.

Reply:
I've removed the getters,  will add them as internal attributes in spec. At beginning I want to get the size of query data by this method when resolving, instead of passing in a actual number.